lib/backend/buffer.go

```diff
@@ -327,6 +355,55 @@ func (w *BufferWatcher) Done() <-chan struct{} {
 	return w.ctx.Done()
 }
 
+// flushBacklog attempts to push any backlogged events into the
+// event channel.  returns true if backlog is empty.
+func (w *BufferWatcher) flushBacklog() (ok bool) {
```

Comment:
nit: named var unused

```suggestion
func (w *BufferWatcher) flushBacklog() bool {
```

Reply:
This is deliberate.  IMO the `(ok bool)` pattern is a useful idiom for distinguishing between bools as "checks" and bools as success/failure indicators.

Reply:
Ack. Fair enough.